tests/auth_tests/test_management.py

```diff
@@ -550,7 +550,6 @@ def test(self):
 
         test(self)
 
-    # TODO: fix this test
     def _test_blank_username(self):
```

Comment:
You need to remove the underscore prefix from the test name to allow the test to run.